src/Router.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
```

Comment:
fix linter errors.
Also review husky, because it should not allow you to commit code with errors

Reply:
I have a warning  `./components/signin/Signin` import should occur before import of `./enums` here